web-app/js/portal/filter/GeometryFilterPanel.js

```diff
@@ -37,8 +52,15 @@ Portal.filter.GeometryFilterPanel = Ext.extend(Portal.filter.BaseFilterPanel, {
         this.add(this.spatialSubsetControlsPanel);
     },
 
-    isVisualised: function() {
-        return false;
+    getFilterData: function() {
+
+        return {
+            name: this.filter.getName(),
+            visualised: this.isVisualised(),
+            cql: this.getCQL(),
+            humanValue: this._getCQLHumanValue(),
+            type: "geom"
```

Comment:
Good question, I hadn't noticed that.

That is something we will need to look at in further refactorings. The value 'geom' here is actually arbitrary, it's not a Portal FilterType and it's not any of the GeoServer types. I'm not sure what we'll do here as if we use the new Filter object's type then there are 5 types that count as geometries so the code (elsewhere where this value is used) would have to check for 5 possible values instead of just this one. We might be able to replace this with a nicely-named boolean (e.g. 'isGeometry: true') or soemthing.